src/docs/content/tutorials/quickstart/index.md

```diff
@@ -1,213 +1,274 @@
 
-In this tutorial, we will use an application named *Computer-Database* deployed at the URL: [http://computer-database.gatling.io](http://computer-database.gatling.io).
+#### Setup the file 
 
-### Scenario
+1. Navigate to `src/tests/java/` and create a new package named `myFirstPackage`.
+2. Create a Java class named `MyFirstScript` inside `myFirstPackage`.
+3. Import the Gatling packages:
 
-To test the performance of this application, we will create scenarios representative of what really happens when users navigate it.
+    ```java
+    package MyFirstPackage;
 
-Here is what we think a real user would do with the application:
+    import static io.gatling.javaapi.core.CoreDsl.*;
+    import static io.gatling.javaapi.http.HttpDsl.*;
 
-1. A user arrives at the application.
-2. The user searches for 'macbook'.
-3. The user opens one of the related models.
-4. The user goes back to home page.
-5. The user iterates through pages.
-6. The user creates a new model.
+    import io.gatling.javaapi.core.*;
+    import io.gatling.javaapi.http.*;
 
-## Basics
+    ```
 
-### Using the Recorder
+#### Extend the `Simulation` class 
 
-To ease the creation of the scenario, we will use the *Recorder*, a tool provided with Gatling that allows you to record your actions on a web application and export them as a Gatling scenario.
+After the import statements, you must extend Gatling's `Simulation` class to write a script. To extend the `Simulation` class add: 
 
-This tool is launched with a script located in the *bin* directory:
+```java
+public class MyFirstScript extends Simulation {
 
-* On Linux/Unix:
 
-```bash
-$GATLING_HOME/bin/recorder.sh
+}
 ```
+#### Define the protocol class
+Inside the `MyFirstScript` class, add an `HTTP protocol` class. Learn about all of the `HttpProtocolBuilder` options in the [Documentation]({{< ref "../../reference/current/http/protocol/index" >}}). For this example, the `baseUrl` property is hardcoded as the Gatling computer database test site, and the `acceptHeader` and `contentTypeHeader` properties are set to `application/json`.  
+
+```java
+public class MyFirstScript extends Simulation {
 
-* On Windows:
+//Add the HttpProtocolBuilder:
 
-```batch
-%GATLING_HOME%\bin\recorder.bat
+HttpProtocolBuilder httpProtocol =
+                http.baseUrl("https://computer-database.gatling.io")
+                  .acceptHeader("application/json")
+                  .contentTypeHeader("application/json");
+
+}
 ```
 
-Once launched, the following GUI lets you configure how requests and responses will be recorded.
+#### Write the scenario
+The next step is to describe the user journey. For a web application, this normally consists of a user arriving at the application and then a series of interactions with the application. For example, if you visit https://computer-database.gatling.io, there is a **Add a new computer** button in the top right corner. The following scenario mocks a user:
```

Comment:
I'm not sure a fake interaction is a good thing for a "Getting started".
We don't allow real data modification (here data addition) because of moderation issues.

Reply:
I agree. That is why I only added the step of clicking the link, not posting any data. What do you recommend? Would a search be better?

Reply:
To be able to add a check, yes